kernel/kernel-api/src/main/java/io/delta/kernel/commit/CommitMetadata.java

```diff
@@ -102,6 +107,11 @@ public CommitInfo getCommitInfo() {
     return commitInfo;
   }
 
+  /** The new {@link DomainMetadata}s that are being written as part of this commit. */
+  public List<DomainMetadata> getNewDomainMetadatas() {
```

Comment:
can we also document it also contains removed domain metadata? (at the first glance, it was had for me to realize this fact.)